src/components/audio-feedback.js

```diff
@@ -196,24 +267,8 @@ const SPRITE_NAMES = {
   ]
 };
 
-export function micLevelForVolume(volume, max) {
-  return max === 0
-    ? 0
-    : volume < max * 0.02
-      ? 0
-      : volume < max * 0.03
-        ? 1
-        : volume < max * 0.06
-          ? 2
-          : volume < max * 0.08
-            ? 3
-            : volume < max * 0.16
-              ? 4
-              : volume < max * 0.32
-                ? 5
-                : volume < max * 0.5
-                  ? 6
-                  : 7;
+export function micLevelForVolume(volume) {
+  return Math.ceil(THREE.Math.mapLinear(volume, 0, 1, 0, 7));
```

Comment:
Wonder if this will feel good enough or if it would feel better if either we hand-wrote this map (which I did before by trying different values until it "felt right" to me) or figuring out how to map the 0-1 in "raw volume" (or... "average square amplitude across all the levels") to "perceived volume"

Reply:
I think we should tune for that at the analyser level, since I think that's what we want everywhere we are reading that data.